package.json

```diff
@@ -19,14 +19,14 @@
     "lint:scripts": "eslint --fix --config .eslintrc.js --ext .jsx,.js,.ts,.tsx scripts",
     "lint:detox": "eslint --fix --config .eslintrc.js --ext .jsx,.js,.ts,.tsx detox",
     "lint:configs": "eslint --fix --config .eslintrc.js --ext .jsx,.js,.ts,.tsx configs",
-    "test": "lerna run test --ignore @mendix/custom-widgets-utils-internal --stream",
+    "test": "lerna run test --ignore @mendix/custom-widgets-utils-internal --ignore data --stream",
     "test:scripts": "lerna run test:scripts --stream",
     "test:scripts:cross-platform": "lerna run test:scripts:cross-platform --stream",
-    "test:e2e": "lerna-parallelism run --ignore @mendix/custom-widgets-utils-internal --stream --concurrency 1 --split 3 test:e2e",
+    "test:e2e": "lerna-parallelism run --ignore @mendix/custom-widgets-utils-internal --ignore data --stream --concurrency 1 --split 3 test:e2e",
```

Comment:
Are you sure is needed? There is no task for `test:e2e` in the `package.json` from scripts/data. I even thing we dont need to ignore at all as test in the package just echo some message.

Reply:
Done